App.js

```diff
@@ -25,9 +26,38 @@ export default function App() {
         setTodos((prev) => prev.filter((todo) => todo.key != key));
     };
 
+    const storeLocally=()=>{
+        _storeData = async () => {
+            try {
+              await AsyncStorage.setItem(
+                'localTodos',
+                JSON.stringify(todos)
+              );
+            } catch (error) {
+              console.log(error);
+            }
+          };
+    }
+    
+
+    useEffect(()=>{
+        try{
+            const value=await AsyncStorage.getItem('localTodos');
+            if(value!=null){
+                let parsedValues=JSON.parse(value);
+                setTodos(parsedValues);
+            }
+        }
+        catch(error){
+            console.log(error);
+        }
+    },[todos]);
```

Comment:
Remove `todos` from the array, and put a blank array instead.

Reply:
Okay